services/keybase/keybase-btc.tester.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const Joi = require('joi')
+
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('existing bitcoin address')
+  .get('/skyplabs.json')
+  .expectJSONTypes(
+    Joi.object({
+      name: 'btc',
+      value: Joi.string(),
```

Comment:
`Joi.string()` will allow 'not found' to get through.

You can use `withRegex(/^(?!not found$)/)` instead to match everything except `not found`.

Alternatively you can use/create something like`withRegex(/^[13][a-km-zA-HJ-NP-Z1-9]{25,34}$/)` which will only match a valid bitcoin address.

Add `withRegex` with `const { withRegex } = require('../test-validators')` at the top of the file.

Reply:
Thanks. It's indeed better now.

Done for BTC.